tests/integration/data/test_storage_provider/dynamic-pvc.yaml

```diff
@@ -5,12 +5,12 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
-  name: raw-block-pvc
+  name: test-pvc
 spec:
   accessModes:
     - ReadWriteOnce
   volumeMode: Filesystem
   resources:
     requests:
       storage: 64Mi
-  storageClassName: ceph-xfs
+  storageClassName: '${storage_class_name}'
```

Comment:
rather than a flat file, let's make this PVC request more dynamic.  This uses python's `string.Template` type to fill in a `storage_class_name` at runtime